core/src/main/scripts/importer/validation_report_template.html.jinja

```diff
@@ -167,6 +166,60 @@
                 {% endif %}
                 <td>{{ record.getMessage()|e }}</td>
                 <td>{{ format_aggregated(record, 'cause') }}</td>
+                {% macro create_modal(filename, record, attr_name, num_record) %}
+                    {% if record[attr_name + '_list'] is defined -%}
+                      {% if (record['show_all_values'] is not defined or record['show_all_values'] == False) and (record[attr_name + '_list']|length) > 3 -%}
+                        <!-- Trigger/Open The Modal -->
+                        {% set num_values = record[attr_name + '_list']|length|string %}
+                        {% set modal_name = "modal" + filename.rsplit('.', 1)[0] + num_record + attr_name %}
+                        <a href="#showValues" data-toggle="modal" data-backdrop="false" data-target=".{{ modal_name }}">See all {{ num_values }} values </a>
+
+                        <div class="modal fade {{ modal_name }}" tabindex="-1" role="dialog" aria-labelledby="myLargeModalLabel">
+                          <div class="modal-dialog modal-lg" role="document">
+                            <div class="modal-content">
+                              <div class="modal-header">
+                                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+                                <h4 class="modal-title" id="gridSystemModalLabel">Listing all values for the message: {{ record.getMessage() }}</h4>
```

Comment:
Aren't you placing the message into the HTML without escaping characters like `<` here? The `record.getMessage()` call above this new macro definition does apply `e` filter, suggesting that the return value is  _not_ supposed to be parsed as HTML.

Reply:
Thanks @fedde-s for spotting this!

Reply:
The problem seems to have been solved for this particular line, but still exists for every other insertion of data from the input file into the HTML code.